autoload/lsp/handlers.vim

```diff
@@ -200,9 +200,17 @@ export def ProcessRequest(lspserver: dict<any>, request: dict<any>)
       'workspace/semanticTokens/refresh': ProcessUnsupportedReq
     }
 
+  # Explicitly ignored requests
+  var lspIgnoredRequestHandlers: list<string> =
+    [
+      # Eclipse java language server sends the 'workspace/executeClientCommand' 
+      # request (to reload bundles) which is not in the LSP specification.
+      'workspace/executeClientCommand',
```

Comment:
not sure if you'd researched `workspace/executeClientCommand` what's that for?
some req msg though not a LSP specification, but that's useful, 
e.g `textDocument/switchSourceHeader` is only for clangd, but we had it.

// but we do truly require some code like this to ignore req msg IF/which it was really not useful.

Reply:
I think `workspace/executeClientCommand` is for Eclipse IDE to reload some bundle with eclipse related packages. Document is scarce. If it turns out to be important then we can provide a callback through #310